test/test_helper.rb

```diff
@@ -43,15 +44,15 @@ def mock_auth_hash(user)
   end
 
   def perform_login(user = nil)
-    user ||= users(:dee)
+    @user ||= users(:dee)
```

Comment:
I think user would be just a variable instead of an instance since it's only being used locally but I'm really bad at differentiating the two so I could 100% be wrong.

Reply:
it doesn't work for me when it's not an instance variable, so i changed it - let me know if you've used it successfully without it